fix/epsilon/ell1gen.fix.d

```diff
@@ -207,6 +208,8 @@ void main(string[] args)
                 "info|i", "Show heap usage information.", &info,
                 "verbose|v", "Print verbose parser error messages.", &verbose,
                 "write|w", "Toggle default for writing output.", &write,
+                "trailingTokens|t", "allowing arbitrary trailing content before EOF (Oberon2 like behavior)", 
```

Comment:
Use consistent term! It's
- ignore content before EOF
- trailing tokens
- training content before EOF

I prefer "content", because both approaches don't use the lexer for the trailing stuff.

(BTW: it's not just Oberon 2, but already Oberon)

Reply:
Not relevant anymore, we go for a backward incompatible change. See below.